library/templates/address_list_form.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+/**
+ * Address List Form
+ *
+ * @package   OpenEMR
+ * @author    David Eschelbacher <psoas@tampabay.rr.com>
+ * @copyright Copyright (c) 2022 David Eschelbacher <psoas@tampabay.rr.com>
+ * @link      http://www.open-emr.org
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ */
+
+use OpenEMR\Services\ContactService;
+
+global $pid; // we need to grab our pid from our global settings.
+
+$contactService = new ContactService();
+$addresses = $contactService->getContactsForPatient($pid);
+
+$table_id = uniqid("table_form_addresses");
+// should always be set, but just in case we will set it to 0 so we can grab it
+$field_id_esc = $field_id_esc ?? '0';
+$addresses = $addresses ?? [];
+
+$name_field_id = "form_" . $field_id_esc;
+$smallform = $smallform ?? '';
+?>
+<div>
+    <table class='form_addresses table table-sm' id="<?php echo attr($table_id); ?>">
+        <thead class ="thead-light">
+        <tr>
+            <th colspan="2">
+                <?php echo xlt("Additional Addresses"); ?>&nbsp;&nbsp;&nbsp;&nbsp;
+                <span class="fas fa-plus-square" style="color:#007bff;" onclick="AddAddress(event);return false"></span>
+            </th>
+        </tr>
+        </thead>
+    </table>
+
+</div>
```

Comment:
I hate to be the ole fuddy-duddy but, our whole purpose of spending all the effort to refactor to bootstrap was for responsiveness. We've gradually been moving our LBF render engine to bootstrap with our eye set to refactor out all the table layouts into templates as plug in and reusable.

These forms are not too difficult to make into bootstrap forms with all the advantages that entails.

I'm not strongly opposed to the current form layouts however we are simply adding more work for later where this project could give us the opportunity to set a pattern for future development.

Reply:
I don't mind refactoring the forms, my primary motivation was first taking what David had and getting it up and working in the quickest way possible and then time willing working on improving the UX.

Reply:
We just have to support multiple pt addr stat.

Reply:
Got all the style changes in to make this responsive.